lib/riffed/client.ex

```diff
@@ -228,7 +228,8 @@ defmodule Riffed.Client do
             {new_client, exception}
           {:ok, rsp} ->
             {new_client, rsp}
-          other = {other, rsp} ->
```

Comment:
ok, so the problem was that `other` is used as the left side & a part of the right side of the match, yeah that's strange. what even would be the result here?

Reply:
Yeah I don't think this ever worked/would raise an error, since it's trying to assign the variable to part and all of the leftside value..